php/commands/sidebar.php

```diff
@@ -73,6 +73,204 @@ public function list_( $args, $assoc_args ) {
 		$formatter->display_items( $wp_registered_sidebars );
 	}
 
+	/**
+	 * Reset one or all sidebars.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<sidebar-id>]
+	 * : ID for the corresponding sidebar.
+	 *
+	 * @subcommand reset
+	 */
+	public function reset( $args, $assoc_args ) {
+		if ( empty( $args ) ) {
+			$this->reset_all();
+		} else {
+			$this->reset_single( $args[0] );
+		}
+	}
+
+	/**
+	 * Reset all sidebars.
+	 */
+	private function reset_all() {
+		global $wp_registered_sidebars;
+
+		if ( ! empty( $wp_registered_sidebars ) ) {
+			foreach ( $wp_registered_sidebars as $sidebar_id => $sidebar ) {
+				if ( 'wp_inactive_widgets' !== $sidebar_id ) {
+					$this->reset_single( $sidebar_id );
+				}
+			}
+		}
+		else {
+			WP_CLI::warning( 'No registered sidebars.' );
+		}
+	}
+
+	/**
+	 * Reset sidebar.
+	 *
+	 * @param string $sidebar_id Sidebar ID.
+	 */
+	private function reset_single( $sidebar_id ) {
+		$this->validate_sidebar( $sidebar_id );
+
+		$widgets = $this->get_sidebar_widgets( $sidebar_id );
+		if ( empty( $widgets ) ) {
+			WP_CLI::warning( sprintf( "'%s' is already empty.", $sidebar_id ) );
+		}
+		else {
+			foreach ( $widgets as $widget ) {
+				$widget_id = $widget->id;
+				list( $name, $option_index, $new_sidebar_id, $sidebar_index ) = $this->get_widget_data( $widget_id );
+				$this->move_sidebar_widget( $widget_id, $new_sidebar_id, 'wp_inactive_widgets', $sidebar_index, 0 );
+			}
+			WP_CLI::success( sprintf( "Sidebar '%s' reset.", $sidebar_id ) );
+		}
+	}
+
+	/**
+	 * Check whether a sidebar is a valid sidebar.
+	 *
+	 * @param string $sidebar_id Sidebar ID.
+	 */
+	private function validate_sidebar( $sidebar_id ) {
```

Comment:
Given this method is only used once, can't we just include its logic in the `reset()` method?